Dockerfile

```diff
@@ -30,9 +27,6 @@ COPY  lib /opt/safe-settings/lib
 USER root
 RUN apk add --no-cache make python
```

Comment:
We don't need to run install python any more. So these 2 lines can be removed. It'll make the Dockerfile simpler